velox/exec/OperatorUtils.cpp

```diff
@@ -182,4 +182,13 @@ wrap(vector_size_t size, BufferPtr mapping, const RowVectorPtr& vector) {
       size,
       wrappedChildren);
 }
+
+void loadColumns(const RowVectorPtr& input, core::ExecCtx& execCtx) {
+  EvalCtx evalCtx(&execCtx, nullptr, input.get());
```

Comment:
Why not just call LazyVector::ensureLoadedRows(input, rows) ? I'm concerned about using EvalCtx which is tightly coupled with expression evaluation outside of that logic.